tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -305,14 +305,22 @@ oe_result_t output_sgx_report(const uint8_t* report, size_t report_size)
     printf("            cpusvn (hex): ");
     oe_hex_dump(report_body->cpusvn, OE_COUNTOF(report_body->cpusvn));
     printf("            miscselect: 0x%x\n", report_body->miscselect);
+    printf("            isvextprodid (hex): ");
+    oe_hex_dump(
+        report_body->isvextprodid, OE_COUNTOF(report_body->isvextprodid));
     printf("            attributes (hex): ");
     oe_hex_dump(&report_body->attributes, sizeof(report_body->attributes));
     printf("            mrenclave (hex): ");
     oe_hex_dump(report_body->mrenclave, sizeof(report_body->mrenclave));
     printf("            mrsigner (hex): ");
     oe_hex_dump(report_body->mrsigner, sizeof(report_body->mrsigner));
+    printf("            configid (hex): ");
+    oe_hex_dump(report_body->configid, sizeof(report_body->configid));
```

Comment:
Why do some fields use `sizeof` while others use `OE_COUNTOF`?

Reply:
Thanks for pointing it out. Will update it.

Reply:
`attributes`, `signature` and `report_data` are not uint8_t[] but a struct. So using `sizeof` for them.